wurst_run.args

```diff
@@ -1,3 +1,5 @@
 -runcompiletimefunctions
 -injectobjects
 -stacktraces
+--hotdoc
```

Comment:
don't commit this

Reply:
Meaning this whole file or just the --hotdoc?

Reply:
whole file

Reply:
`-noPjass` is still needed for running the demo map i believe.
Code should compile without?